API.md

```diff
@@ -74,20 +74,20 @@ Here's the complete rundown of how files and directories are mapped to API calls
 >  - vision - ≥2 and <5
 
 #### Path prefix
-> [`server.path(relativeTo)`](https://github.com/hapijs/hapi/blob/master/API.md#serverpathrelativeto)
+> [`server.path(relativeTo)`](https://github.com/hapijs/hapi/blob/master/API.md#server.path())
```

Comment:
The other option is to point to the `v16` branch, which I think will stick around for a while.

Reply:
Thanks for thinking of the links!  Good call!

Reply:
ahh!! very cool. I had no idea that was hanging around, thanks for pointing that out. I'll update 🙏